openmetadata-ui/src/main/resources/ui/src/pages/DatabaseSchemaPage/DatabaseSchemaPage.component.tsx

```diff
@@ -670,6 +684,70 @@ const DatabaseSchemaPage: FunctionComponent = () => {
     );
   };
 
+  const handleShowDeletedTables = (checked: boolean) => {
+    setShowDeletedTables(checked);
```

Comment:
Don't we need to hit the API to get the latest data

Reply:
@chirag-madlani  we are hitting the API by keeping the `showDeletedTables`  in the dependency of useEffect of API.